xbmc/video/dialogs/GUIDialogVideoManager.h

```diff
@@ -66,6 +75,7 @@ class CGUIDialogVideoManager : public CGUIDialog
   std::unique_ptr<CFileItemList> m_videoAssetsList;
   std::shared_ptr<CFileItem> m_selectedVideoAsset;
   bool m_hasUpdatedItems{false};
+  std::map<ManagerMessage, unsigned int> m_messages;
```

Comment:
Could this be an `unordered_map` ?

Reply:
I read conflicting information about the performance of map vs unordered_map. With such a small number of entries, might as well make it an array, using the enum values as index.